superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -973,7 +981,8 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
 
     notificationSettings.forEach(setting => {
       if (!!setting.method && setting.recipients?.length) {
-        hasInfo = true;
+        hasInfo =
+          setting.method === 'Email' ? !!setting.email_subject?.length : true;
```

Comment:
I think this is causing [the valid report unit test](https://github.com/apache/superset/blob/d6dbb95cc45360ce24778811ab5a6d5f9a4198e6/superset-frontend/src/features/alerts/AlertReportModal.test.tsx#L255) to fail. Might be causing your CI error. Do you need to push `email_subject` to `settings` in `onNotificationAdd`?